BlazarBase/src/main/java/com/hubspot/blazar/base/DependencyItem.java

```diff
@@ -0,0 +1,60 @@
+package com.hubspot.blazar.base;
+
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.MoreObjects;
+
+public class DependencyItem {
```

Comment:
Maybe just `Dependency`? Other things that come to mind are `Package` or `Artifact`